src/components/MainApp/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Form } from 'react-bootstrap';
+import { InputNewTodo } from '../InputNewTodo';
+import UserSelect from '../UserSelect';
+import { connect } from 'react-redux';
+import styles from './MainApp.module.css';
+
+
+type Todo = {
+    title: string,
+    user?: number,
+    isDone: boolean,
+}
```

Comment:
this type definition should be extracted outside into it's own file, could be placed inside:
```
src/models/Todo.ts
or
src/types/Todo.ts
```